lua/weapons/weapon_ghost_ak47.lua

```diff
@@ -1,3 +1,5 @@
+AddCSLuaFile()
```

Comment:
Because this way is dumb.
And the function of this PR would be broken without using AddCSLuaFile() manually.